packages/terra-graphs-docs/NOTICE

```diff
@@ -0,0 +1,13 @@
+Copyright 2020 Cerner Innovation, Inc.
```

Comment:
```suggestion
Copyright 2021 - present Cerner Innovation, Inc.
```

Reply:
I think there is a line like this in the readme that should also be updated.

Reply:
Similar to this: https://github.com/cerner/terra-toolkit/blob/main/packages/jest-config-terra/README.md#license